build.gradle

```diff
@@ -14,30 +14,38 @@ repositories {
     mavenCentral()
 }
 
-def grpcVersion = '1.51.0'
-def protobufVersion = '3.21.12'
+def grpcVersion = '1.60.2'
+def protobufVersion = '3.25.3'
 def slf4jVersion = '1.7.36'
-def log4jVersion = '2.19.0'
+def log4jVersion = '2.23.1'
+def picoCliVersion = '4.7.5'
+def jakartaJsonVersion = '1.1.6'
+def jakartaAnnotationVersion = '1.3.5'
+def dnsJavaVersion = '2.1.9'
+def junitVersion = '4.12'
+def mockitoVersion = '2.16.0'
+def assertJVersion = '3.9.1'
+def protoGoogleCommonProtosVersion = '2.36.0'
+def googleapisCommonProtosVersion = '0.0.3'
 
 dependencies {
     implementation "org.slf4j:slf4j-api:${slf4jVersion}"
     implementation "org.apache.logging.log4j:log4j-core:${log4jVersion}"
     implementation "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
-    implementation 'info.picocli:picocli:4.0.1'
-    implementation 'javax.json:javax.json-api:1.1.4'
-    implementation 'org.glassfish:javax.json:1.1.4'
```

Comment:
this package is replaced by `jakarta.json`
https://mvnrepository.com/artifact/org.glassfish/javax.json